pkg/kubernetes/kubernetes.go

```diff
@@ -34,7 +34,7 @@ import (
 // Client represents a set of helper methods for interacting with the
 // Kubernetes API.
 type Client struct {
-	client *kubernetes.Clientset
```

Comment:
Is this just random cleanup?

Reply:
looks like it's being used in controller-manager PR, and seems to be a good change, as I stumbled upon it a while ago: hard to use "regular" APIs if you wrap your k8s client with `Client`

Reply:
This allows users of this package to get all the convenience functions we have added plus all the underlying Kubernetes functions.